doc/modules/clustering.rst

```diff
@@ -350,6 +352,13 @@ to be the exemplar of sample :math:`i` is given by:
 
 To begin with, all values for :math:`r` and :math:`a` are set to zero,
 and the calculation of each iterates until convergence.
+As discussed above, in order to avoid numerical oscillations when updating the 
+messages, the damping factor :math:`\lambda` is introduced to iteration process:
+
+.. math:: r_{t+1}(i, k) = \lambda\cdot r_{t}(i, k) + (1-\lambda)\cdot r_{t+1}(i, k)
```

Comment:
So to be sure, this means the previous value has to contribute at least half of the new value relative to the incoming message, but can contribute more. Yes?